package.json

```diff
@@ -15,10 +15,14 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@chakra-ui/react": "^1.0.0",
+    "@emotion/react": "^11.1.1",
+    "@emotion/styled": "^11.0.0",
     "axios": "^0.19.2",
     "express": "^4.17.1",
+    "framer-motion": "^2.9.4",
     "morgan": "^1.10.0",
```

Comment:
We know you did not add this, but it would be best to move this to a dev dependency...